my-first-project/src/app/heroes/heroes.component.ts

```diff
@@ -16,6 +16,14 @@ export class HeroesComponent implements OnInit {
     name: 'Windstorm'
   };
 
+  heroes = HEROES;
+
+
+  selectedHero: Hero;
+  onSelect(hero: Hero): void{
```

Comment:
We usually leave a new line between attributes and functions or functions themselves.

We also like to append public/private modifiers to each attribute and function. In this case they are all public